automata/pda/configuration.py

```diff
@@ -27,3 +28,16 @@ def __repr__(self) -> str:
         return "{}({!r}, {!r}, {!r})".format(
             self.__class__.__name__, self.state, self.remaining_input, self.stack
         )
+
+    def __eq__(self, other: Any) -> bool:
+        """Return True if two PDAConfiguration are equivalent"""
+        if not isinstance(other, PDAConfiguration):
+            raise NotImplemented
```

Comment:
@Vipul-Cariappa `return` should be used here instead of `raise`:

```suggestion
            return NotImplemented
```

Fore more details on why this is, see this StackOverflow post: [Why return NotImplemented instead of raising NotImplementedError](https://stackoverflow.com/questions/878943/why-return-notimplemented-instead-of-raising-notimplementederror)

Reply:
I have made the suggested changes.